syntax = "proto3";

import "google/protobuf/any.proto";

package services;

message Task {
    uint32 task_id = 1;
    repeated google.protobuf.Any payload = 2;
}